docs/routers/nokia-G240WB.md

```diff
@@ -0,0 +1,92 @@
+This guide was developed using a `Nokia G-240W-B` router (software version 3FE47969AGCA21). It should work for other models in the `G` series.
+
+!!! note
+    There is no single way to do it right. Chose the one best suited for you.
+This guide is `IPv4` and `IPv6` compatible, but it is not mandatory to use `IPv6`.
+
+### Prerequisites
+
+- The `Raspberry Pi` is accessible from the `LAN` network via `SSH`.
```

Comment:
Why did you remove the information about a static IP (or fixed DHCP assignment)?

Reply:
I dont recall why, maybe I misunderstood your last comment. Adding back that section.